pkg/webhook/kubernetes/sync_jobs.go

```diff
@@ -0,0 +1,114 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package webhook_k8s
+
+import (
+	"context"
+	"github.com/devfile/devworkspace-operator/internal/images"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func SyncJobToCluster(
```

Comment:
It seems to be free from TLS or webhook server logic. Maybe then it makes sense to move to some common place.